src/sentry/grouping/fingerprinting/__init__.py

```diff
@@ -466,3 +493,38 @@ def visit_key(self, node, children):
     def visit_quoted_key(self, node, children):
         # leading ! are used to indicate negation. make sure they don't appear.
         return node.match.groups()[0].lstrip("!")
+
+
+def _load_configs():
+    if not CONFIGS_DIR.exists():
+        logger.warning(
+            "Failed to load Fingerprinting Configs, invalid _config_dir: %s",
+            CONFIGS_DIR,
+        )
+
+    configs = {}
+
+    for config_file_path in sorted(CONFIGS_DIR.glob("**/*.txt")):
+        config_name = config_file_path.parent.name
+        configs.setdefault(config_name, [])
+
+        try:
+            with open(config_file_path) as config_file:
+                str_conf = config_file.read().rstrip()
+                configs[config_name].extend(FingerprintingRules.from_config_string(str_conf).rules)
+        except InvalidFingerprintingConfig as ex:
+            logger.warning(
+                "Fingerprinting Config %s Invalid: %s",
+                config_file_path,
+                ex,
+            )
+        except Exception as ex:
```

Comment:
@vartec if loading fails, would it make more sense to just re-raise the exception and stop Sentry from booting up?

Reply:
@scefali Not sure if I'd want it to completely blow up in prod just because it couldn't load one file. But I can add re-raise `if settings.DEBUG` so it blows up in CI.